crates/storage/src/lib.rs

```diff
@@ -7,8 +7,7 @@ pub mod hyperlane_messages;
 
 // every storage module should implement this trait
 pub trait Storage {
-    fn default() -> Result<HyperlaneMessageStore>;
-    fn from_env() -> Result<HyperlaneMessageStore>;
+    fn from_path_relative(crate_depth: usize) -> Result<HyperlaneMessageStore>;
```

Comment:
I'm fine to merge this for now but I don't think we should include APIs like this in traits. 

Any test fixtures of db's etc should be separate from what we consider code which will eventually go to prod